docs/5.advanced/3.monitoring.md

```diff
@@ -78,25 +78,45 @@ prometheus: {}
 
 The following metrics are exposed under `dipdup` namespace:
 
-| metric name                                    | description                                    |
-| ---------------------------------------------- | ---------------------------------------------- |
-| `dipdup_indexes_total`                         | Number of indexes in operation by status       |
-| `dipdup_index_total_sync_duration_seconds`     | Duration of the last synchronization           |
-| `dipdup_index_total_realtime_duration_seconds` | Duration of the last realtime queue processing |
-| `dipdup_index_levels_to_sync_total`            | Number of levels to reach synced state         |
-| `dipdup_index_levels_to_realtime_total`        | Number of levels to reach realtime state       |
-| `dipdup_index_handlers_matched_total`          | Index hits for each handler                    |
-| `dipdup_datasource_head_updated_timestamp`     | Timestamp of the last head update              |
-| `dipdup_datasource_rollbacks_total`            | Number of rollbacks                            |
-| `dipdup_http_errors_total`                     | Number of HTTP errors                          |
+| metric name                                    | description                                         | type      |
```

Comment:
I think I'm going to write a script like `scripts/docs.py dump-demos` to dump this table automatically, so don't spend time on it